storage/innobase/trx/trx0rseg.cc

```diff
@@ -89,7 +89,7 @@ trx_rseg_update_wsrep_checkpoint(
 
 	if (xid_seqno != -1
 	    && !memcmp(xid_uuid, wsrep_uuid, sizeof wsrep_uuid)) {
-		ut_ad(xid_seqno > wsrep_seqno);
+		ut_ad(xid_seqno >= wsrep_seqno);
```

Comment:
This assertion should not be changed. It is a sanity check for server side transaction commit processing to catch situations where a transaction is committed twice or out of order. If this assertion fires, the problem is most likely in the server side code.

Reply:
Due rolling upgrade problem (10.3 -> 10.4) if GTID is not set (server_id==0) we need to reinitialize it and store it permanently into SE. This will trigger assert because same galera seqno is used twice
Issue is also later comment, will check for more appropriate solution.

Reply:
Removed as this is not needed anymore due changes in earlier initialization of gtid state.